client/html/main.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel = "stylesheet" href = "../css/main.css">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title> WRITE YOUR WISH TO MAKE IT HAPPEN </title>
+  </head>
+  <body>
+    <header>
+      <div class="container" id="user">
+        <p id="hi">
+          <span id=user_name>a</span>'s list /  <a id="linkA" href="/logout">Sign out</a>
+        </p>
+      </div>
+    </header>
+    <div class="container" id="main-container" >
+      <div id="row1" class="row">
+        <div class="col-2" id="today"><label for="today">Create Date</label></br></br>20210502</div>
+        <div class="container col-8" id="form">
+          <form class="form-inline" action="/html/main.html" method="post" name="frm" id="user-signin" accept-charset="UTF-8">
+            <div class="inline" id="duedate">
+              <label for="due">Due Date</label>
+              <input class="form-control" type="text" id="due" name="due">
+            </div>
+            <div class="inline" id="wishtodo">
+              <label for="todo">Wish / Todo</label>
+              <input class="form-control" type="text" id="todo" name="todo" required>
```

Comment:
In general, I think you rely on `id`s too much.  Classes should be used a lot more in all of your HTML files.

Reply:
I see your point.
The reason why is I am implementing too detail and variety of action or attribute to each components.
I will make things simpler and keep your point in mind for the next tiime.